File: gui/src/main/java/nisui/gui/NisuiGuiLauncher.java
package nisui.gui;

import java.io.InputStream;
import java.io.PrintStream;

import nisui.core.NisuiAppLauncher;
import nisui.core.NisuiFactory;

public class NisuiGuiLauncher implements NisuiAppLauncher {
    @Override
    public void run(NisuiFactory nisuiFactory, InputStream in, PrintStream out) {
        MainFrame mainFrame = new MainFrame(nisuiFactory, in, out);
        mainFrame.setVisible(true);
    }
}

File: gui/src/main/kotlin/nisui/gui/NisuiGuiLauncher.kt
package nisui.gui

import java.io.InputStream
import java.io.PrintStream

import nisui.core.NisuiAppLauncher
import nisui.core.NisuiFactory

public class NisuiGuiLauncher: NisuiAppLauncher {
    override fun run(nisuiFactory: NisuiFactory, sin: InputStream, sout: PrintStream) {
        val mainFrame = MainFrame(nisuiFactory, sin, sout);
        mainFrame.setVisible(true);
    }
}
